### DISCUSSION TITLE
Order Creation Job is not working

### DISCUSSION THREAD

**Discussion by a-akashsoni** [Kyon147/laravel-shopify#199] [Q&A]
I have installed the shopify package with Laravel 8, and I have created a order creation webhook using job. But when I implemented the code and execute, order creation job is not working. Kindly help me to resolve the issue so I use the order creation Job and execute my project.

Kindly revert as soon as possible.

Thanks

**Comment by sumantaakeans2020** [comment]
Same issue i am also getting i am using ng-fork for my local but web hook not coming, any help will be appriciated

**Reply by tariqbilal** [reply]
I am facing the same issue can you discuss it here?

https://github.com/Kyon147/laravel-shopify/issues/232

**Comment by matthewhaworth** [comment]
You likely need to go to API access against your app and allow it to access customer private data, otherwise you'll be getting this error silently: "This app is not approved to subscribe to webhook topics containing protected customer data.".

This module does not handle errors that are returned from GraphQL for failed webhooks subscriptions currently.
